Database-Files/CreateDatabase.sql

```diff
@@ -0,0 +1,128 @@
+DROP SCHEMA IF EXISTS kJourneyDB;
```

Comment:
Can you attach a screenshot of this script running without errors?

Reply:
![image](https://user-images.githubusercontent.com/44677467/68292634-3306fb00-0084-11ea-9139-05cc44c17e78.png)

Reply:
looks good